ui/src/app/pages/workspace/invalid-billing-banner-maybe.tsx

```diff
@@ -217,7 +217,8 @@ export const InvalidBillingBannerMaybe = fp.flow(
   const { creatorUser } = workspace || {};
   const [showExtensionModal, setShowExtensionModal] = React.useState(false);
   const isCreator = profile?.username === workspace?.creatorUser?.userName;
-  const isEligibleForExtension = profile?.eligibleForInitialCreditsExtension;
```

Comment:
Before this change, eligibility was based on the current user, but that is not accurate. A workspace being eligible for an extension is determined by the workspace creator.

Reply:
thanks for catching this!